src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -130,6 +131,10 @@ def prepare(self, corpus_dir, target_path, build_dir):
       environment.set_value('USE_EXTRA_SANITIZERS', False)
 
     arguments.extend(strategy_info.arguments)
+    # Update strategy info with environment variables from fuzzer's options.
+    for env_var_name, value in extra_env:
```

Comment:
You need a None check here, per the test failure.

Reply:
Fair enough, done :-)